### ISSUE TITLE
The "setup" section of iCloud sync is rarely ticked

### ISSUE THREAD

**Issue by wadjj** [onmyway133/PastePal#17]
**Describe the bug**
Unless I clear everything from iCloud and get started from scratch, it almost always show this in the settings

**To Reproduce**
I just normally use it, over time (a couple of days), it will always come to and stay like this, the download and upload are fine, but setup is not in sync.
Besides, I don't know what exactly does "setup" mean here.

**Expected behavior**
everything stays in sync

**Screenshots**
<img width="674" alt="image" src="https://github.com/onmyway133/PastePal/assets/7797428/8900efda-aa43-4956-af48-2a440387bdb7">

 - OS: [e.g. iOS] all(Mac,iOS,iPadOS)
 - Version latest(2.14.3 (158))

**Comment by WillFulmer** [comment]
Same issue here

![IMG_3649](https://github.com/onmyway133/PastePal/assets/36053224/2f2952e5-561b-4260-b307-42b7c0346417)

**Comment by WillFulmer** [comment]
And seeing the same on the MacOS side as well:

![image](https://github.com/onmyway133/PastePal/assets/36053224/975cf049-012c-46ce-ac02-79034d1d9d50)

**Comment by javentous** [comment]
I have the same here, it also does not sync from my MacBook → iPhone

**Comment by mrxoom1274** [comment]
Same here. iPhone and iPad sync directly and perfect. MacBook not. Work sometimes and sometimes not.

**Comment by javentous** [comment]
@mrxoom1274 it's never been a problem for you though, right? I haven't had it be an issue yet, though I'm concerned it may be at the crucial point when I need it...

**Comment by mrxoom1274** [comment]
@javentous absolutely. Sync is bad. If I need to check for each copy, if the sync runs or not: then the app is not useful for me !

**Comment by mrxoom1274** [comment]
Settings - sync on Mac https://github.com/IndieGoodies/PastePal/assets/70346648/3262fe56-04d8-4920-90cf-733463f8b2a6

**Comment by drnola** [comment]
I'm still having the setup button unticked issue. Has this been solved by anyone?

**Comment by ejstrates** [comment]
Also having the issue on one mac, where the other mac checks the setup sync
